ProblemsSolved.js

```diff
@@ -0,0 +1,76 @@
+function ProblemsSolved() 
+{
+}
+
+ProblemsSolved.prototype.SumOfEvenNumberInArray = function(array)
+{
+    var sum = 0;
+    for(i=0; i < array.length; i++)
+    {
+		if(array[i]%2 == 0)
+			sum += array[i];
+    }
+    return sum;
+}
+
+ProblemsSolved.prototype.GetTheLargestTwoNumber = function(array)
+{
+    var pairNumbers = [];
+    pairNumbers[0] = -Infinity;
+    pairNumbers[1] = -Infinity;
+	
+	var localMaximum = -Infinity;
+	var indexOfMaximumNumber = 0;
+	
+	if(array.length < 2)
+		return 'Array length is to small. Give an array with minimum 2 pairNumbers';
```

Comment:
Inconsistent return type for function (string and array of numbers). 
Instead of returning string when preconditions are not met, consider "throw"-ing.